src/modules/billing/mappers/api/invoice.js

```diff
@@ -28,7 +28,9 @@ const map = {
  * @return {Object}
  */
 const modelToBatchInvoice = invoice => {
-  return objectMapper(invoice, map);
+  const mappedInvoice = objectMapper(invoice, map);
+  objectMapper.setKeyValue(mappedInvoice, 'isMinimumChargeApplied', invoice.isMinimumChargeApplied());
```

Comment:
Is this property needed in the UI list view?
If so, could this be added to the `map` above?

Reply:
I was unable to accomplish it in the map, however, I no longer use it in the UI, so I have removed it